src/components/ListItem.js

```diff
@@ -8,6 +11,30 @@ import { userShoppingList } from '../lib/shoppingListsCollection';
 import calculateEstimate from '../lib/estimates';
 
 const ListItem = ({ listItem, itemId }) => {
+  const removeItem = () => {
+    const isOk = window.confirm(
+      `Are you sure you want to delele ${listItem.name}?`,
+    );
+    if (isOk) {
+      userShoppingList()
+        .update({
+          [itemId]: firebase.firestore.FieldValue.delete(),
+        })
+        .then((res) => {
+          Bus.emit('flash', {
+            content: `${listItem.name} deleted successfully`,
+            type: 'success',
+          });
+        })
+        .catch((res) => {
+          Bus.emit('flash', {
+            content: `Something went wrong, Sorry!!!`,
+            type: 'error',
+          });
```

Comment:
I didn't know about `EventEmitter` and `events` so I had to google them. However, I loved that you used them to show different types of messages to the user. ❤️